spec/proto/cmd/bnsd/x/qualityscore/codec.proto

```diff
@@ -10,14 +10,14 @@ message Configuration {
   // This defines the Address that is allowed to update the Configuration object and is
   // needed to make use of gconf.NewUpdateConfigurationHandler
   bytes owner = 2 ;
-  float c = 3;
-  float k = 4;
-  float kp = 5;
-  float q0 = 6;
-  float x = 7;
-  float x_inf = 8;
-  float x_sup = 9;
-  float delta = 10;
+  weave.Fraction c = 3 ;
```

Comment:
s/ ;/;/

Reply:
This is autogenerated, I have no control over this.